src/Styles/index.css

```diff
@@ -23,31 +24,80 @@
     color: black;
 }
 
-.fade{
-    height: 749px;
-    background: linear-gradient(70deg, #7FFFD4, #9932CC);
-    z-index: 0;
-    position: relative;
+.background{
+    background: #F5F5F5;
+}
+
+.background32{
+    background: red;
+    width: 100%;
+    height: 810px;
+
 }
 
 .header-scrolled{
-    background:white;
+    background:#C0C0C0;
     width: 100%;
     height: 80px;
     position: fixed;
     color: black;
     z-index: 5;
 }
 
+.about{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: white;
+    height: 700px;
+    width: 990px;
```

Comment:
Don't set the dimensions of the block. Just set the paddings and let it determine its own size